webots_ros2_examples/setup.py

```diff
@@ -6,8 +6,10 @@
 data_files = []
 data_files.append(('share/ament_index/resource_index/packages', ['resource/' + package_name]))
 data_files.append(('share/' + package_name, ['launch/example.launch.py']))
-data_files.append(('share/' + package_name + '/worlds',
-                   ['worlds/ros_example.wbt', 'worlds/.ros_example.wbproj']))
+data_files.append(('share/' + package_name + '/worlds', [
+    'worlds/ros_example.wbt', 'worlds/.ros_example.wbproj',
+    'worlds/tiago++_example.wbt', 'worlds/.tiago++_example.wbproj'
```

Comment:
it's a bit strange to have a tiago example in this package as we have a dedicated tiago package.

Reply:
It makes sense, I will move it